src/amo/components/RatingsByStar/index.js

```diff
@@ -108,17 +116,20 @@ export class RatingsByStarBase extends React.Component<InternalProps> {
             let starCount;
             let starCountNode;
 
-            function createLink(text) {
+            function createLink(text, count, tab = true) {
               invariant(addon, 'addon was unexpectedly empty');
 
+              const attributes = tab ? {} : { 'aria-hidden': true };
```

Comment:
Actually, aria-hidden isn't an HTML boolean attribute. It's a string attribute with the value "true" or "false". (I think this is because ARIA isn't part of HTML itself, so it can't be specified as a real boolean?)

Does this get translated to `aria-hidden="true"` in the output or just `aria-hidden`? If the latter, you'll need to make this change:

```suggestion
              const attributes = tab ? {} : { 'aria-hidden': "true" };
```

Reply:
Ah thanks for catching that, that's pretty confusing, but probably explains why it was handled correctly by react.

Reply:
I think that means the update wasn't required because it came out as a string in the markup as that was what I was looking at when I made the original comment.

Reply:
@muffinresearch So, Should I revert the last commit or just commit the above suggestion to stringify that bool value?

Reply:
I think you can remove the last commit as that was ok as it was. Apologies for pointing you in the wrong direction.

Reply:
No problem!